Apps/Common/src/Services/EmailQueueService.cs

```diff
@@ -128,19 +128,20 @@ public void CloneAndQueue(Guid emailId, bool shouldCommit = true)
         }
 
         /// <inheritdoc />
-        public void QueueNewInviteEmail(string hdid, string toEmail, Uri activationHost)
+        public void QueueNewInviteEmail(string hdid, string toEmail, Uri activationHost, int verificationExpiryHours)
         {
             Dictionary<string, string> keyValues = new Dictionary<string, string>();
             MessagingVerification invite = new MessagingVerification();
             invite.InviteKey = Guid.NewGuid();
             invite.HdId = hdid;
-            invite.ExpireDate = DateTime.UtcNow.AddDays(VerificationExpiryDays);
+            invite.ExpireDate = DateTime.UtcNow.AddHours(verificationExpiryHours);
 
             string hostUrl = activationHost.ToString();
             hostUrl = hostUrl.Remove(hostUrl.Length - 1, 1); // Strips last slash
 
-            keyValues.Add(INVITE_KEY_VARIABLE, invite.InviteKey.ToString());
+            keyValues.Add(INVITE_KEY_VARIABLE, hdid);
             keyValues.Add(ACTIVATION_HOST_VARIABLE, hostUrl);
+            keyValues.Add(VERIFICATION_EXPIRY_HOURS, $"{verificationExpiryHours}");
```

Comment:
.ToString() instead of $"{}"

Reply:
@tiagograf-AOT can you explain why .ToString() is better than $"{}" here?

Reply:
readability

Reply:
:-) not sure about that though, mbe just our personal preference as I see `keyValues.Add(VERIFICATION_EXPIRY_HOURS, $"{verificationExpiryHours}");` is readable as well

Reply:
The ${} syntax implies that you are doing some string operations. So when reading it we need to make sure that only the stuff we want is being made a string.
The .ToString() means that the value is being transformed into a string, no need for further checking if the string is being modified.

TlDr: ${} Is good for string composition ToString() is good for value=> string.